ansible/roles/csm.rr.mgmt_nodes_placement_validation/files/mgmt_nodes_placement_validation.py

```diff
@@ -0,0 +1,257 @@
+
+    if worker_nodes_cnt >= min_worker_node_cnt:
+        if rack_cnt >= min_rack_cnt and worker_nodes_cnt >= min_worker_node_cnt and  missing_cnt > node_rack_missing_quorum:
+            if rack_cnt == min_rack_cnt and worker_nodes_cnt == min_worker_node_cnt
+                print(bcolors.FAIL + "\nOne or more racks missing worker node under minimum 3 rack condition" + bcolors.ENDC)
+                print(bcolors.FAIL + "\nPlease re arrange the worker nodes for equal distribution" + bcolors.ENDC)
+                print(bcolors.FAIL + "Exiting placement validation...\n"  + bcolors.ENDC)
+            elif rack_cnt > min_rack_cnt and worker_nodes_cnt >= min_worker_node_cnt
+                print(bcolors.FAIL + "\nMore than one rack missing worker node in a", rack_cnt , "rack system" + bcolors.ENDC)
+                print(bcolors.FAIL + "\nPlease re arrange the worker nodes for equal distribution" + bcolors.ENDC)
+                print(bcolors.FAIL + "Exiting placement validation...\n"  + bcolors.ENDC)
+            sys.exit(1)
+    else:
+        print(bcolors.FAIL + "\nRack Resiliency can not be enabled with", worker_nodes_cnt, "worker nodes" + bcolors.ENDC)
+        print(bcolors.FAIL + "\nNeed minimum of 3 worker nodes configured" + bcolors.ENDC)
+        print(bcolors.FAIL + "Exiting placement validation...\n"  + bcolors.ENDC)
+        sys.exit(1)
+
+def validate_ceph_nodes_placement(placements_dict):
+    rack_cnt=1
+    storage_nodes_cnt=0
+    missing_cnt=0
+
+    print(bcolors.OKGREEN + bcolors.UNDERLINE + "\nValidate Management Storage (CEPH) nodes"+ bcolors.ENDC)
+
+    for rack_cnt, (rack_id, nodes) in enumerate(placements_dict.items()):
+        print("\nRack number: ", rack_cnt)
+        print("Rack ID: ", rack_id)
+        print("Management Nodes: ", nodes)
+
+        found=0
+        for node in nodes:
+            if re.match(r"^.*ncn-s00[0-9]$", node):
+                # Found storage node in a rack
+                storage_nodes_cnt+=1
+                found=1
+
+        if found == 0:
+            # Missing storage node in a rack
+            missing=1
+            missing_cnt+=1
+
+        rack_cnt+=1
+
+    print("\nTotal number of storage nodes: ", storage_nodes_cnt)
+
+    node_rack_missing_quorum=get_node_rack_missing_quorum(rack_cnt)
+
+    if storage_nodes_cnt >= min_storage_node_cnt:
+        if rack_cnt == min_rack_cnt and storage_nodes_cnt == min_master_node_cnt and  node_rack_missing_quorum >= node_rack_missing_quorum:
+            print(bcolors.FAIL + "\nOne or more racks missing storage node under minimum 3 rack condition" + bcolors.ENDC)
+            print(bcolors.FAIL + "\nPlease re arrange the storage nodes for equal distribution" + bcolors.ENDC)
+            print(bcolors.FAIL + "Exiting placement validation...\n"  + bcolors.ENDC)
+            sys.exit(1)
+        elif rack_cnt > min_rack_cnt and storage_nodes_cnt >= min_master_node_cnt and node_rack_missing_quorum > node_rack_missing_quorum:
```

Comment:
The previous branch ends with `sys.exit(1)`, so this can just be an `if`
```suggestion
        if rack_cnt > min_rack_cnt and storage_nodes_cnt >= min_master_node_cnt and node_rack_missing_quorum > node_rack_missing_quorum:
```